ledger/src/blockstore_db.rs

```diff
@@ -992,7 +992,7 @@ pub struct Database {
     column_options: Arc<LedgerColumnOptions>,
 }
 
-#[derive(Debug, Clone)]
```

Comment:
Why this struct should implement `Clone` to begin with?
I think we can remove it.

Reply:
Thanks for spotting that.  I think it does not require Clone.  Let me remove it.